support/debugtracer_amd64.go

```diff
@@ -0,0 +1,16 @@
+//go:build amd64 && debugtracer
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Apache License 2.0.
+ * See the file "LICENSE" for details.
+ */
```

Comment:
There's currently [another PR](https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/146) in flight that gets rid of these copyright notices. Let's not add new ones here.